modules/Core/pages/panel/privacy_and_terms.php

```diff
@@ -100,15 +100,17 @@
 // Get privacy policy + terms
 $site_terms = DB::getInstance()->get('privacy_terms', ['name', 'terms'])->results();
 if (!count($site_terms)) {
-    $site_terms = DB::getInstance()->get('settings', ['name', 't_and_c_site'])->results();
+    $site_terms = Util::getSetting('t_and_c_site');
+} else {
+    $site_terms = $site_terms[0]->value;
 }
-$site_terms = $site_terms[0]->value;
 
 $site_privacy = DB::getInstance()->get('privacy_terms', ['name', 'privacy'])->results();
 if (!count($site_privacy)) {
-    $site_privacy = DB::getInstance()->get('settings', ['name', 'privacy_policy'])->results();
+    $site_privacy = Util::getSetting('privacy_policy');
+} else {
+    $site_privacy = $site_privacy[0]->value;
 }
```

Comment:
I feel like these are likely for backwards compatibility which would have been initially made for years ago... probably safe to always assume `privacy_policy` and `t_and_c_site` (or whichever of the two we are currently setting during install for each of them)?

Reply:
Yes let's get rid of this read from the settings table since we now use the separate `privacy_terms` table